tasks-dev/assemble.js

```diff
@@ -3,9 +3,8 @@ const assemble = require('assemble');
 const helpers = require('handlebars-helpers');
 const debug = require('gulp-debug');
 const plumber = require('gulp-plumber');
-const newer = require('gulp-newer');
+// const newer = require('gulp-newer');
```

Comment:
I left it in because in the comment below it sounds like it may come back in the near future. I'm happy to remove it though?